README.md

```diff
@@ -83,27 +79,80 @@ The core set of binding data is:
 * **password** - the password or token used to log into the service.  Can be omitted if no authorization required, or take another format such as an API key.  It is strongly recommended that the corresponding ConfigMap metadata properly describes this key.
 * **certificate** - the certificate used by the client to connect to the service.  Can be omitted if no certificate is required, or simply point to another Secret that holds the client certificate.  
 * **uri** - for convenience, the full URI of the service in the form of `<protocol>://<host>:<port>/<name>`.
```

Comment:
Where is the value of `<name>` coming from in `<protocol>://<host>:<port>/<name>` for `uri` field?

Reply:
good catch - we changed that to `<type>`, and then forgot to add a `<contextRoot>` key.  Added now.